### PR TITLE
Fix/image title attr

### PR DESCRIPTION
**Pull request by vitorflg**
#### What problem is this solving?

It was lefting some steps to finish adding the title attribute to Image/ImageList/ImageSlider components in the image/link anchors

#### How to test it?

You can go to CMS -> SiteEditor -> ImageList and change the title attribute of the images/links. Now, in the worskpace, you will have to inspect image/link tag in the ImageList section(the last) to check if the attribute was updated!

https://vitorflg--storecomponents.myvtex.com/admin/cms/site-editor
https://vitorflg--storecomponents.myvtex.com/

#### How does this PR make you feel? [:link:](https://media.giphy.com/media/l4KhPbIIDgO3sMw0w/giphy.gif)

Image

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
-  "admin/editor.image-list.images.description.title": "Alt image description",
+  "admin/editor.image-list.images.description.title": "Image alt text",
+  "admin/editor.image-list.images.title.title": "Image title",
   "admin/editor.image-list.images.link.url.title": "Image link URL",
   "admin/editor.image-list.images.link.openNewTab.title": "Should open on new tab",
-  "admin/editor.image-list.images.link.noFollow.title": "Shuold be a nofollow link",
-  "admin/editor.image-list.images.link.title.title": "Title for the image",
-  "admin/editor.image-list.images.title": "Images",
+  "admin/editor.image-list.images.link.noFollow.title": "Should be a nofollow link",
+  "admin/editor.image-list.images.link.title.title": "Title for the link",
```
Comment (klzns) [inline messages/en.json:18]: This is not being used, is it?

**Review by klzns — CHANGES_REQUESTED** [review]
Missing changelog

See https://github.com/vtex-apps/store-image/pull/24#issuecomment-685828485

File: messages/en.json (hunk 1 above)
Comment (vitorflg) [reply messages/en.json:18]: I think not but I don't have 100% sure. Do you think we can remove that(All link messages)?

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-image",
-  "version": "0.5.0",
+  "version": "0.5.1",
```
Comment (kaisermann) [inline manifest.json]: No need to update it manually. The release bot will do that for you!

File: react/Image.tsx [hunk 3]
```diff
@@ -119,7 +119,7 @@ function Image(props: ImageProps) {
       href={formatIOMessage({ id: link.url, intl })}
       rel={link.noFollow ? 'nofollow' : ''}
       target={shouldOpenLinkInNewTab ? '_blank' : undefined}
-      title={formatIOMessage({ id: link.title, intl })}
+      title={formatIOMessage({ id: link?.attributeTitle, intl })}
```
Comment (kaisermann) [inline react/Image.tsx:122]: What happens if there's no `attributeTitle`?

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- `title` prop to `Image` component
```
Comment (kaisermann) [inline CHANGELOG.md:11]: Isn't this an addition? 
```suggestion
### Added
- `title` prop to `Image` component
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (vitorflg) [reply CHANGELOG.md:11]: Already exists a log with the Added mark (## [0.4.0] - 2020-02-10). The props was there but it wasn't being passed correctly

File: manifest.json (hunk 2 above)
Comment (vitorflg) [reply manifest.json]: hmmm, thanks :)

File: react/Image.tsx (hunk 3 above)
Comment (vitorflg) [reply react/Image.tsx:122]: No prop will be passed to Image

File: messages/en.json (hunk 1 above)
Comment (vitorflg) [reply messages/en.json:18]: @klzns

File: react/Image.tsx (hunk 3 above)
Comment (vitorflg) [reply react/Image.tsx:122]: @kaisermann

File: manifest.json (hunk 2 above)
Comment (kaisermann) [reply manifest.json]: Don't forget to remove this change from your branch

File: messages/en.json (hunk 1 above)
Comment (klzns) [reply messages/en.json:18]: If there's no reference to this string in this project, yes

**Review by klzns — APPROVED** [review]

**Review by iago1501 — APPROVED** [review]
For me, it's ok... all the specific modifications were made. Good job

**Pull request merged**